.github/workflows/kuksa_dds_feeder.yml

```diff
@@ -36,27 +36,27 @@ jobs:
         path: ${{github.workspace}}
 
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v1
+      uses: docker/setup-buildx-action@v2
 
     - name: Login to GitHub Container Registry
-      uses: docker/login-action@v1
+      uses: docker/login-action@v2
       with:
         registry: ghcr.io
         username: ${{ github.repository_owner }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
     - id: repository-name-adjusted
       name: Make repository name in lower case for docker upload.
-      uses: ASzc/change-string-case-action@v2
+      uses: ASzc/change-string-case-action@v5
```

Comment:
By the way - for which repository name do we need to change case? This action seems to be used in some *.yml files but not all. Needed?